taxonomy/forms.py

```diff
@@ -0,0 +1,33 @@
+"""
+Taxonomy forms
+"""
+
+from django.forms import (
+    ModelForm,
+    TextInput,
+    CheckboxSelectMultiple,
+    RadioSelect,
+)
+
+from .models import Vocabulary
+
+
+class VocabularyForm(ModelForm):
```

Comment:
I'm referring to the text below each field on this screenshot, for example "Describe how content editors..." under the description textarea. Would help_text render in that way?

![Screenshot](https://cloud.githubusercontent.com/assets/125561/7683265/d3d7ef08-fd4b-11e4-9e84-3db25554030a.png)